### DISCUSSION TITLE
Wiki Rendering Stuck Forever

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [requarks/wiki#4304] [Error / Bug Report]
If I leave the wiki running for a few days, it stops being able to successfully render. Any rendering job will save, but the "rendering" box does not go away, and then eventually gets terminated by the reverse proxy. Refreshing the page will show the new content. However, a simple restart fixes the problem. I still think running a cronjob to restart the server is "hacky".

**Comment by Sergeus** [comment]
This is old, but I'm seeing the same behavior. Even without a fix - it'd be good to know where to look for more information.

**Comment by ajf8729** [comment]
Seeing similar myself. Haven't looked into it all to much yet, but I did bump the resources on the host VPS because it was pretty small, but the issue came back, but seems to be a bit longer (maybe 2 weeks) until it and then a reboot "fixes" it.
